content/root/best-practice-guide.textile

```diff
@@ -0,0 +1,146 @@
+---
+title: Best Practice Guide
+section: root
+index: 30
+jump_to:
+  Contents:
+    - Ably 101
+    - Using Ably like a pro
+---
+
+h2(#title). Ably Best Practice Guide
+
+This best practice guide aims to quickly get you to understand the best way to make use of Ably's Realtime platform for your applications. These recommendations include some important considerations that are otherwise prone to easy ignoration while skimming through the docs. 
+
+This guide consists of the following two sections:
+* Ably 101 - Key concepts that you must understand before moving on to the best practice recommendations.
+* Using Ably like a pro - Actual recommendations that we've compiled as per our experience with our users, in order for you to avoid common mistakes or misunderstandings whilst using Ably.
```

Comment:
Why "Actual" for recommendations.  Actual makes it sound like other recommendations are not.

Perhaps: "Recommendations that we've compiled from our experience with developers to help you avoid common mistakes or misunderstandings"